service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -196,8 +197,8 @@ private ApiWorkspaceDescription buildWorkspaceDescription(
         .stage(workspace.getWorkspaceStage().toApiModel())
         .gcpContext(gcpContext)
         .azureContext(azureContext)
-        .lastUpdatedDate(
-            workspaceActivityLogDao.getLastUpdatedDate(workspace.getWorkspaceId()).orElse(null));
+        .lastUpdatedDate(workspaceActivityLogDao.getLastUpdatedDate(workspaceUuid).orElse(null))
```

Comment:
It would be more efficient to do these calculations as a single transaction in SQL and a single request to the DAO. However, if we are going to move the created date into the workspace row eventually, it would not be worth the trouble.

Reply:
My understanding of end state was: we're moving this to middle layer, and not adding a new column.

WSM GetWorkspace no longer returns created on, last updated

We add WSM GetActivityLog

Middle layer calls GetWorkspace, GetActivityLog, and returns a workspace with created on, last updated

(If this turns out to be slow, we could add column in the future)

Reply:
+1 for melissa's thinking for having an api to fetch the activity log per workspace since we eventually want trail. 

But even with a middle layer, maybe we don't *always* want the whole logs and singling out created on and last updated can improve performance (??). Then getting the two in one trip sounds good to me